### DISCUSSION TITLE
Help with manifest - Photos Legacy

### DISCUSSION THREAD

**Discussion by MegaSheepMaster** [microsoft/winget-pkgs#95194] [Q&A]
Hello,

Sorry if this sounds dumb but how do i get the manifest from an app I need added to Winget? I'm need to install Photos Legacy but this app is hidden in the MS Store. You can only access it by navigating to Photos - Settings then Get Photos Legacy.

Thanks in advance

**Comment by mdanish-kh** [comment]
A manifest can only be added for a package in WinGet Packages Repository for which you can get an InstallerUrl from the official source. For the Photos Legacy App, WinGet does show this app as available from the MSSTORE source if we use it's ID i.e., `winget show 9NV2L4XVMCXM`, but I can't seem to install it through `winget install 9NV2L4XVMCXM`. You might want to track this issue related to this app in the winget-cli repo https://github.com/microsoft/winget-cli/issues/2770

**Reply by MegaSheepMaster** [reply]
Thanks but this seems to be a dead end too. Sadly I already opened a ticket with MS and they directed me here lol tossing the problem somewhere else I guess...

**Reply by denelon** [reply]
The actual restriction seems to be coming from the Microsoft Store "msstore" source. I'm not seeing it in the output from a query there.
